test/jdk/java/lang/instrument/RetransformRecordAnnotation.java

```diff
@@ -29,7 +29,6 @@
  * @library /test/lib
  * @run shell MakeJAR.sh retransformAgent
  * @run main/othervm -javaagent:retransformAgent.jar -Xlog:redefine+class=trace RetransformRecordAnnotation
- * @run main/othervm -javaagent:retransformAgent.jar -XX:+PreserveAllAnnotations -Xlog:redefine+class=trace RetransformRecordAnnotation
```

Comment:
This test is described as:
```
 * @summary test that records with invisible annotation can be retransformed
 ```
which suggests to me the test can actually be deleted as it serves no purpose now there are no invisible annotations

Reply:
Agree.
Removed the test.